doc/guide.md

```diff
@@ -115,6 +116,12 @@ There are several approaches to securely operating and exposing `code-server`.
 
 **tip**: You can list the full set of `code-server` options with `code-server --help`
 
+### iPad Simple Setup
+
+Now that you have an exposed code-server instance with an IP and port number, you are one simple step away.
```

Comment:
What's the point of this app vs just visiting code-server with safari?

Reply:
So you know how the documentation says iPads don’t work easy and you need a cloud or self signed ssl injection for iPads to support copy/paste and other commands....

Ya, you don’t need any of that with Serveditor.

With this app you don’t have to cloud setup or config a secure certificate setup for a full featured Visual Studio.

This has everything to do with iOS and its https requirements of a true ssl

Reply:
Wouldn't it just be over `http` then at that point? Or does it access code-server over `ssh`?

I've got self signed certificates working on iPad now! See #2255

Will be in next release.

Reply:
Whatever protocol it does use, it just works including copy paste and additional features that code server does NOT support unless on https